### DISCUSSION TITLE
Is there a similar function as "lsp--buffer-content" in lsp-mode?

### DISCUSSION THREAD

**Discussion by zhenhua-wang** [joaotavora/eglot#1029] [Q&A]
In lsp-mode, I can control the content sent to the server using "lsp--buffer-content". This is quite useful for polymode, as it can let me ignore the markdown parts. Is there a similar function in eglot?

**Comment by joaotavora** [comment]
I don't understand what you want to do, maybe someone else does.

Imagine this function existed: what would be its signature and what would you do with it?
